### PR TITLE
fix monthly build

### PR DESCRIPTION
**Pull request by rkpagadala**
fix monthly build, 

1) the docker images were expected to be tagged as istio/pilot:TAG, but were actually tagged as gcr.io/pilot:TAG. Fix by reading the actual tag and retag as required.
2) use the docker image from istio-testing instead of istio-io as done by all other build scripts
3) make gsutil quiet

### CODE REVIEW

File: release/publish_release.sh [hunk 1]
```diff
@@ -227,7 +227,7 @@ fi
 if [[ "${DO_GCS}" == "true" ]]; then
   echo "Copying to GCS destination ${GCS_DEST}"
   gsutil -m cp "${UPLOAD_DIR}"/deb/istio*.deb "gs://${GCS_DEST}/deb/"
-  gsutil -m cp "${UPLOAD_DIR}/SHA256SUMS"     "gs://${GCS_DEST}/"
+# gsutil -m cp "${UPLOAD_DIR}/SHA256SUMS"     "gs://${GCS_DEST}/"
```
Comment (sebastienvas) [inline release/publish_release.sh]: why are we removing this ?

Comment (rkpagadala) [reply release/publish_release.sh]: unfinished, will finish and add it in a subsequent PR

File: release/cloud_publish.template.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "steps": [
     {
-      "name": "gcr.io/istio-io/istio-builder:0.5.7",
+      "name": "gcr.io/istio-testing/istio-builder:0.5.7",
```
Comment (hklai) [inline release/cloud_publish.template.json]: are we retiring the use of istio-io project?

Comment (rkpagadala) [reply release/cloud_publish.template.json]: no, we are not, istio-builder is being pushed to istio-testing as part of our normal update for this docker image. We frequently forget to push to istio-io, also some other release scripts use istio-testing container. Reducing confusion by using a single source.

**Pull request merged**